indexer/feature_data.cpp

```diff
@@ -471,6 +473,10 @@ bool FeatureParams::FinishAddingTypes()
   if (m_Types.size() > kMaxTypesCount)
     m_Types.resize(kMaxTypesCount);
 
+  // Patch fix that removes house number from localities.
```

Comment:
Это разве не баг в осме?

> On 12.4.2016, at 19.55, Viktor Govako <notifications@github.com> wrote:
> 
> In indexer/feature_data.cpp:
> 
> > @@ -471,6 +473,10 @@ bool FeatureParams::FinishAddingTypes()
> >    if (m_Types.size() > kMaxTypesCount)
> >      m_Types.resize(kMaxTypesCount);
> >  
> > +  // Patch fix that removes house number from localities.
> Точка Париж въодит в состав relation, за которым записано addr:housenumber.
> И так еще кучка городов в Европе.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
>